presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -67,6 +67,12 @@ Resource Group Properties
 * ``jmxExport`` (optional): If true, group statistics are exported to JMX for monitoring.
   Defaults to ``false``.
 
+* ``queuedTimeLimit`` (optional): maximum amount of time a query can get queued in this group before
```

Comment:
How about *can get queued in* -> *may be queued by*

Reply:
I assume this means the maximum queued time limit is the sum of the limits for all the groups a query will use.